<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextViewGUI</class>
 <widget class="QDialog" name="TextViewGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>721</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Text View</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>350</width>
         <height>350</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="exportBox" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Target:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="targetComboBox">
            <item>
             <property name="text">
              <string>Forum</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>SL</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_numbering">
            <property name="text">
             <string>Move numbering</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="cb_coords">
            <property name="text">
             <string>Coordinates</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="Horizontal Spacing3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="buttonToClip">
            <property name="toolTip">
             <string>Copy position to clipboard</string>
            </property>
            <property name="text">
             <string>&amp;To clipboard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonRefresh">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonSave">
            <property name="toolTip">
             <string>Save position to file</string>
            </property>
            <property name="text">
             <string>&amp;Save</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="buttonClose">
            <property name="toolTip">
             <string>Close this window</string>
            </property>
            <property name="text">
             <string>&amp;Close</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="gtpBox" native="true">
       <layout class="QHBoxLayout" name="gtpLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="leftMargin">
         <number>1</number>
        </property>
        <property name="topMargin">
         <number>1</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>1</number>
        </property>
        <item>
         <spacer name="Horizontal Spacing2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAbort">
          <property name="toolTip">
           <string>Abort program startup</string>
          </property>
          <property name="text">
           <string>&amp;Abort</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
